tsk/auto/db_postgresql.cpp

```diff
@@ -125,11 +118,16 @@ PGconn* TskDbPostgreSQL::connectToDatabase(TSK_TCHAR *dbName) {
     PQescapeString(&userName_sql[0], userName, strlen(userName));
     PQescapeString(&password_sql[0], password, strlen(password));
     PQescapeString(&hostName_sql[0], hostNameOrIpAddr, strlen(hostNameOrIpAddr));
+#ifdef TSK_WIN32
+	snprintf(connectionString, 1024, "user=%s password=%s dbname=%S host=%s port=%s", userName_sql, password_sql, dbName, hostName_sql, hostPort);
```

Comment:
Can't we use the %PRI.... for this?

Reply:
It works fine on linux and then it complains on windows. So, only I added the ifdef logic.